tests/tasks/test_dao_complete_task.py

```diff
@@ -37,6 +38,19 @@ def test_achieve_not_existent_task(self):
 
         self.assertEqual(expected_response, actual_response)
 
+    def test_achieve_task_not_in_relation(self):
+        task_id = 1
+
+        self.assertFalse(self.tasks_list_2.find_task_by_id(task_id).get("is_done"))
+        expected_response = (
+            messages.MENTORSHIP_RELATION_DOES_NOT_EXIST,
```

Comment:
this test is about achieving task that does not exist in relation, but the message is about mentorship relation not existing 🤔 @devkapilbansal

Reply:
Yes, @isabelcosta This test checks if a mentorship relation exists or not before trying to delete a task.
Is something missing or wrong here. :confused:

Reply:
Ah ok, so the name of the test could be improved to reflect that.
`test_delete_task_from_non_existing_relation` something like that 🤔

Reply:
Done @isabelcosta